graylog2-web-interface/src/pages/ShowDashboardPage.jsx

```diff
@@ -131,8 +134,8 @@ const ShowDashboardPage = React.createClass({
       positions[widget.id] = {
         col: (persistedDimensions.col === undefined ? defaultDimensions.col : persistedDimensions.col),
         row: (persistedDimensions.row === undefined ? defaultDimensions.row : persistedDimensions.row),
-        height: (persistedDimensions.height === undefined ? defaultDimensions.height : persistedDimensions.height),
-        width: (persistedDimensions.width === undefined ? defaultDimensions.width : persistedDimensions.width),
+        height: (this._validDimension(persistedDimensions.height) ? persistedDimensions.height : defaultDimensions.height),
```

Comment:
For me this is by far more explicit, as I'm calling a function just to say this may have a different value than just `undefined`.